plugins/tweak/tweaks/embark-profile-name.h

```diff
@@ -1,22 +1,21 @@
 #include "df/viewscreen_setupdwarfgamest.h"
-using namespace DFHack;
+
 struct embark_profile_name_hook : df::viewscreen_setupdwarfgamest {
     typedef df::viewscreen_setupdwarfgamest interpose_base;
-    DEFINE_VMETHOD_INTERPOSE(void, feed, (std::set<df::interface_key> *input))
-    {
+
+    DEFINE_VMETHOD_INTERPOSE( void, feed, ( std::set<df::interface_key> *input ) ) {
```

Comment:
Could you maintain the style here? Specifically, we usually don't usually use spaces immediately inside parentheses, but do after `if`/`for`/etc. (e.g. `if (x)`). Per https://docs.dfhack.org/en/stable/Contributing.html, opening braces on their own line are acceptable (in fact, that used to be the *only* style mentioned there, so you'll see a lot of older DFHack code using it). I don't have a strong preference about the braces personally, but I'd rather keep what's already here.